woocommerce/woocommerce
# Remove Unnecessary React Admin Build

### All Submissions:






### Changes proposed in this Pull Request:

Since we are using Nx, there's no reason to manually rebuild the React admin and assets for WooCommerce in its `"build"` script. Running `pnpm nx run build woocommerce` will already do that before building the Core plugin.

### How to test the changes in this Pull Request:

1. Run `rm -rf plugins/woocommerce/assets && git reset --hard`.
2. Make sure there are no built `plugins/woocommerce/assets/client`
3. Run `pnpm nx build woocommerce --skip-nx-cache`.
4. Verify that there are now assets in `plugins/woocommerce/assets/client` 

### Other information:




### FOR PR REVIEWER ONLY:
